index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home</title>
+</head>
+
+<body>
+    <section class="home-section">
+        <header class="header1">
+            <div class="header1-content">
+                <ul>
+                   <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </header>
+        <header>
+            <img src="img/menu.png" width="40" height="40" alt="hamburger-button" class="hamburger-button">
+            <img src="img/close.png" width="40" height="40" alt="close-btn" class="close-btn">
+            <nav class="navbar" id="navbar-list">
+                <ul>
+                    <li id="logo"><a href="index.html" rel="noreferrer noopener"><img src="img/logo.jpg" alt="logoimage" width="130" height="50"></a></li>
+                    <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Program</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Join</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+                    <li><a href="#" rel="noreferrer noopener">News</a></li>
+                    <li id="websummit"><a href="#" rel="noreferrer noopener">Web Summit 2021</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="main-title">
+            <h2>"Hello,Tech World"</h2>
+            <h1>Technology Conference<br>Web Summit<br>2021</h1>
+        </div>
+        <div class="description">
+            <p>Web Summit brings together the people and companies redefining the global tech industry.
+                Web Summit will return to Lisbon this year.</p>
+        </div>
+        <div class="date-time">
+            <h2>November 1-4, 2021</h2>
+            <h3>@ ALTICE ARENA & FIL, LISBON</h3>
+        </div>
+        <div class="startup-program">
+            <h2>Startup Program</h2>
```

Comment:
- [ ] The text inside columns should be more white in order to look like the original
![Screenshot from 2021-10-13 19-47-25](https://user-images.githubusercontent.com/62705947/137186087-c778a871-a02e-4c8a-b237-95b3977698d9.png)